archetypes/archetypes/src/main/archetype/common/files/.gitignore.mustache

```diff
@@ -15,7 +15,9 @@ target/
 *.rar
 
 # IntelliJ Idea
-.idea/
+{{#gitignore-intellij}}
```

Comment:
Use `!.idea/runConfigurations` instead. See https://git-scm.com/docs/gitignore
Also, use a generic model variable for the `.gitignore`, it does not have to be specific to IntelliJ.